ArduCopter/mode_plancktracking.cpp

```diff
@@ -53,6 +53,40 @@ void ModePlanckTracking::run() {
             relative_angle);
     }
 
+    //Check for tether high tension
+    bool high_tension = copter.planck_interface.is_tether_high_tension() || copter.planck_interface.is_tether_timed_out();
+    bool use_positive_throttle = high_tension;
+    if(high_tension) {
+        //If we ever enter high tension, make sure ACE is landing
+        if((copter.flightmode != &copter.mode_planckland) && (copter.flightmode != &copter.mode_planckrtb) && copter.planck_interface.ready_for_land()) {
+            copter.set_mode_planck_RTB_or_planck_land(ModeReason::GCS_FAILSAFE);
```

Comment:
What if it's not in planck land or RTB, and is not ready for land? Shouldn't we still change to RTB?

Reply:
I'm not sure we want to do that in the event of comms loss. If you transitioned to RTB without being ready for land, it would return to the last known location of the CBGPS

Reply:
Ah, good point. That seems to leave the possibility of being in high tension but not trying to land, but I guess that's ok, since it will be in pos throttle